components/modal/modalDialog/Modal.razor.cs

```diff
@@ -213,8 +213,7 @@ private string GetWidth()
         /// <summary>
         /// Is RTL
         /// </summary>
-        [CascadingParameter]
-        public bool Rtl { get; set; } = false;
+        public bool Rtl => base.RTL;
```

Comment:
You can directly use `base.RTL`.

Reply:
> You can directly use `base.RTL`.

When using the template, it need to get RTL from the outside, but base.RTL is protected

Reply:
That's fine.